recipes/julia-snail

```diff
@@ -0,0 +1,5 @@
+(julia-snail
+ :fetcher github
+ :repo "gcv/julia-snail"
+ :files ("*"
```

Comment:
Could this recipe be simplified to use the `:defaults` recipe with one additional `:include`?  One reason for this is to make it easier for your package to grow without requiring additional changes to the recipe.
```elisp
(julia-snail
 :fetcher github
 :repo "gcv/julia-snail"
 :files (:defaults "JuliaSnail.jl"))
```
(I'm not sure if you also need snail.png, and generally MELPA doesn't distribute the LICENSE and README)

Reply:
Thanks for all the comments, @riscy. I believe I resolved all of them with the latest batch of commits to the julia-snail repository. I also simplified the recipe as per your suggestion.